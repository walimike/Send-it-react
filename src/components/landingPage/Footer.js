import React, { Component } from "react";

class Footer extends Component {
  render() {
    return (
      <div className="page-footer custom">
        <div className="footer-copyright">
          <div className="container center">
            <div className="row">
              <i className="socicon-facebook px-2" />
              <i className="socicon-twitter px-2" />
              <i className="socicon-instagram px-2" />
            </div>
            Copyright Â© 2019 walimike. All rights reserved.
          </div>
        </div>
      </div>
    );
  }
}

export default Footer;
