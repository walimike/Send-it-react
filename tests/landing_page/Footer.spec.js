import React from "react";
import { shallow } from "enzyme";

import Footer from "../../src/components/landingPage/Footer";

describe("Footer", () => {
  it("should render without crashing", () => {
    const wrapper = shallow(<Footer />);
    expect(wrapper).toMatchSnapshot();
  });
});
