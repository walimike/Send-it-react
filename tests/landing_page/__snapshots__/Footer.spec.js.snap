// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer should render without crashing 1`] = `ShallowWrapper {}`;
